src/extension/rn-extension.ts

```diff
@@ -31,8 +35,11 @@ export function activate(context: vscode.ExtensionContext): void {
                     warnWhenReactNativeVersionIsNotSupported();
                     entryPointHandler.runFunction("debugger.setupLauncherStub", "Failed to setup the stub launcher for the debugger", () =>
                         setupReactNativeDebugger()
-                            .then(() => setupReactDir(context))
-                            .then(() => setupExtensionServer(context)));
+                            .then(() =>
+                                setupAndDispose(new ReactDirManager(), context))
```

Comment:
For consistency should we create the ReactDirManager at the same time/scope as the ExtensionServer?